client/src/sockets/enums.js

```diff
@@ -46,6 +46,8 @@ export const ListEvents = Object.freeze({
   REMOVE_MEMBER_ROLE_SUCCESS: ListActionTypes.REMOVE_MEMBER_ROLE_SUCCESS,
   REMOVE_MEMBER_SUCCESS: ListActionTypes.REMOVE_MEMBER_SUCCESS,
   REMOVE_OWNER_ROLE_SUCCESS: ListActionTypes.REMOVE_OWNER_ROLE_SUCCESS,
+  RESTORE_AND_REDIRECT: 'list/RESTORE_AND_REDIRECT',
```

Comment:
why not in `ListActionTypes`?

Reply:
beacuse this is specific to socket and we don't need action type for this.
The same approach is in the line 43 and 45 in this file.
Check this out

Reply:
Because it is used only by socket's events, but to be more consist I'll move it to actions' types (it will not hurt them in any way). @marekrozmus which approach should we use?

Reply:
Ultimately, I think you should have a single list of actions/events that work across sockets/redux-stores and it doesn't matter where and how they are dispatched.